hack/e2e_collect.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Fetch graph data from Cincinnati
+# Input params:
+# * 1 - cincinnati URL
+# * 2 - prefix
+
+type -f curl > /dev/null || {
+  echo Please install curl
+  exit 1
+}
+
+test -z "${1}" && echo "Usage: ./e2e-collect.sh CINCINNATI_URL" && exit 1
+
+TMPFILE="$(mktemp -u --tmpdir=.)"
```

Comment:
Local dir would be the easiest place to find those. `tmp.*` files in `hack/` were added to gitignore so they won't be commited

Reply:
When using `mktemp` we don't care about finding them, as we just want a transient file with a unique name. Since the script emits the path I don't see why it needs to live and be ignored in the workdir.

Reply:
Okay, using standard tmpdir now